openfecli/commands/plan_rbfe_network.py

```diff
@@ -82,9 +84,18 @@ def plan_rbfe_network_main(
     help=OUTPUT_DIR.kwargs["help"] + " Defaults to `./alchemicalNetwork`.",
     default="alchemicalNetwork",
 )
+@click.option(
+    "--radial", required=False, default=False, type=bool, is_flag=True,
+    help="Toggle on generation of a radial network, otherwise uses minimal-spanning-tree",
+)
+@click.option(
+    "--radial_hub", required=False, default=None, type=str,
+    help="For radial networks, the name of the ligand to use as the center",
+)
```

Comment:
Could make sense, to implement these options in parameters and only import them here.

like MOL_DIR, that way you could share the implementation for rhfes an rbfes  and both clis have the same guaranteed cli experience.